src/NetAnalyzers/Core/AnalyzerReleases.Unshipped.md

```diff
@@ -1,5 +1,10 @@
 ; Please do not edit this file manually, it should only be updated through code fix application.
 
+### New Rules
+Rule ID | Category | Severity | Notes
+--------|----------|----------|-------
+CA1839 | Performance | Info | PreferDictionaryContainsMethods, [Documentation](https://docs.microsoft.com/dotnet/fundamentals/code-analysis/quality-rules/ca1839)
```

Comment:
This ID is already in use now, please use a new ID.
https://github.com/dotnet/roslyn-analyzers/blob/release/6.0.1xx-preview4/src/NetAnalyzers/Core/AnalyzerReleases.Unshipped.md

Reply:
Changed to `CA1841`